packages/odyssey-design-tokens/src/border.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
These should be `px` so that they don't scale with type changes.

Reply:
rem units also make sense because If we eventually have a scale for border we want all the widths to update based on a base value for the border. the same thing goes for stuff like radius, they should remain isolated but still scale upon making a change to one of their values.

Reply:
@josesolano-okta practical example:

Our border width is currently `1px`. This is true regardless of a base font-size of 14, 16, or 18px. I assume we do not want that width to get thicker or thinner based on the font size. 

At any size larger than 16px, this would jump to a 2px border. At any size less, it would be sub-pixel.